visidata/main.py

```diff
@@ -103,7 +104,7 @@ def main_vd():
         vd.warning(e)
 
     warnings.showwarning = vd.warning
-    builtins.print = lambda *args, **kwargs: vd.status(*args)
+    builtins.print = functools.wraps(builtins.print)(lambda *args, **kwargs: vd.status(*args))
```

Comment:
Let's add functions `vd.printout` and `vd.printerr` that do this (or `print_stdout` and `print_stderr`) and then use those in preference to `print.__wrapped__` in the actual code.